app/routes/missions/swift.mdx

```diff
@@ -276,6 +276,24 @@ https://swift.gsfc.nasa.gov/archive/
   </table>
 </div>
 
+**JSON-Serialized GCN Notice Types in GCN Kafka:**
+
+The GUANO system distributes alerts and localizations for sub-threshold detections of gamma-ray transients around the times of other triggers.
+These are bursts that did not result in any of the automatic GCN Notices described above, and is the result of analysis after the data arrives on the ground.
+
+These notices are published on the GCN Kafka topic **swift.bat.guano**.
+[Detailed Description and Examples](/schema/gcn/notices/swift/bat/guano/alert.schema.json)
```

Comment:
For now, since we are still working on #894, it would be better to link directly to the directory in the GitHub project view: https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices/swift/bat/guano